tests/reporter/kubernetes.go

```diff
@@ -612,3 +663,14 @@ func getNodesWithVirtLauncher(virtCli kubecli.KubevirtClient) []string {
 
 	return nodes
 }
+
+func writeStringToFile(filePath string, data string) error {
+	f, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return fmt.Errorf("failed to open the file %s: %v", filePath, err)
```

Comment:
it would print the error twice, here and on `fmt.Fprintf(os.Stderr, "%v", err)`
is it ok ?